src/icons/ColorfulEye.tsx

```diff
@@ -0,0 +1,141 @@
+import classnames, { fill, strokeWidth } from 'classnames/tailwind'
+
+const fillClasses = classnames(
+  fill('fill-primary-dimmed'),
+  strokeWidth('stroke-1.5')
+)
+
+export default function ({ open }: { open?: boolean }) {
+  return open ? (
+    <svg
+      width="24"
+      height="17"
+      viewBox="0 0 24 17"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M23 8.5C23 8.5 18.875 16 12 16C5.125 16 1 8.5 1 8.5C1 8.5 5.125 1 12 1C18.875 1 23 8.5 23 8.5Z"
+        stroke="url(#paint0_linear_57_19973)"
+        className={fillClasses}
+      />
+      <ellipse
+        cx="11.999"
+        cy="8.49991"
+        rx="3.23529"
+        ry="3.40909"
+        stroke="url(#paint1_linear_57_19973)"
+        className={fillClasses}
+      />
+      <defs>
+        <linearGradient
+          id="paint0_linear_57_19973"
+          x1="4.3"
+          y1="8.5"
+          x2="23"
+          y2="8.5"
+          gradientUnits="userSpaceOnUse"
+        >
+          <stop stop-color="var(--secondary)" />
+          <stop offset="1" stop-color="var(--accent)" />
+        </linearGradient>
+        <linearGradient
+          id="paint1_linear_57_19973"
+          x1="9.73426"
+          y1="8.49991"
+          x2="15.2343"
+          y2="8.49991"
+          gradientUnits="userSpaceOnUse"
+        >
+          <stop stop-color="var(--secondary)" />
+          <stop offset="1" stop-color="var(--accent)" />
+        </linearGradient>
+      </defs>
+    </svg>
+  ) : (
+    <svg
+      width="22"
```

Comment:
The width differs, this means that the box size changes on click a little, let's make both eyes have same sizes (you an wrap them into div if you need)